cmd/dataset_tokenizer/sanitizer.go

```diff
@@ -43,7 +43,7 @@ func (runeReader SanitizedRuneReader) nextBuffer() *bytes.Buffer {
 		} else if size == 0 {
 			// No valid rune, and we have stuff in our accumulator, so let's
 			// flush and finish up.
-			text = string((*acc)[:*idx])
+			text = string((*acc)[:idx])
```

Comment:
What's this change?

Reply:
wouldn't compile because idx isn't a pointer but the other idx i was changing it from was, so this was just a bug with the prev commit